docs/tutorials/11_deuteron_binding_energy.ipynb

```diff
@@ -0,0 +1,1374 @@
+    "from qiskit_nature.converters.second_quantization import QubitConverter\n",
+    "from qiskit_nature.runtime import VQEClient"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7fff1d3a",
+   "metadata": {},
+   "source": [
+    "## Step 2: Defining some functions that we will use"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fc54fdc6",
+   "metadata": {},
+   "source": [
+    "Before we start the definition of the Deuteron Hamiltonian, we need to define a function that performs the behavior of a mathematical function, the Kronecker delta function, which has the following behavior:\n",
+    "\n",
+    "$\\delta_{nm} = \\bigg\\{\\begin{array}{c}0, \\ \\textrm{if} \\ n \\neq m \\\\1, \\ \\textrm{if } \\ n = m.\\end{array}$.\n",
+    "\n",
+    "This function appears in the Deuteron Hamiltonian definition given in [1]. Below we have a piece of code that defines the Kronecker delta function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "af3784b6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def kronecker_delta_function(n: int, m: int) -> int:\n",
+    "    \"\"\"A function that performs the behavior of a Kronecker delta function.\n",
+    "\n",
+    "    Args:\n",
+    "        n (int): The first integer index of Kronecker delta function.\n",
+    "        m (int): The second integer index of Kronecker delta function.\n",
+    "\n",
+    "    Returns:\n",
+    "        Returns 1 if n = m, else returns 0.\n",
+    "    \"\"\"\n",
+    "    if type(n) == int and type(m) == int:\n",
+    "        if n == m:\n",
+    "            return 1\n",
+    "        else:\n",
+    "            return 0\n",
+    "    else:\n",
+    "        raise ValueError(\n",
+    "            \"The function expects two integer numbers ('int') as input, but were given a {} type and a {} type as input.\".format(\n",
+    "                type(n), type(m)\n",
+    "            )\n",
+    "        )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "553b3feb",
+   "metadata": {},
+   "source": [
+    "In [1] we can find the following expression for the Deuteron Hamiltonian\n",
+    "$H_N = \\sum_{n,n'=0}^{N-1}\\langle n'|(T+V)|n\\rangle a_{n'}^\\dagger a_n$,\n",
+    "where $|n\\rangle$ and $|n'\\rangle$ represent quantum states in the harmonic oscillator basis. With that in mind, the operators $a_n^\\dagger$ and $a_n$ represent the operators of creation and annihilation.\n",
+    "\n",
+    "To create the code that defines $H_N$, we need the matrix elements of the kinetic and potential energy. These expressions we can find in [1] written in the following way\n",
+    "$\\langle n'|T|n\\rangle = \\frac{\\hbar\\omega}{2}\\left[\\left(2n+\\frac{3}{2}\\right)\\delta_{nn'}-\\sqrt{n(n+\\frac{1}{2})}\\delta_{nn'+1}-\\sqrt{(n+1)(n+\\frac{3}{2})}\\delta_{nn'-1}\\right],$\n",
+    "$\\langle n'|V|n\\rangle = V_0\\delta_{n0}\\delta_{nn'}.$\n",
```

Comment:
I suggest that you use `m` instead of `n'` as the apostrophe does not render
very nicely in the documentation.